BioArchLinux/rstudio-desktop/PKGBUILD

```diff
@@ -0,0 +1,138 @@
+makedepends=('git' 'cmake>=3.1.0' 'boost' 'desktop-file-utils' 'jdk8-openjdk' 'apache-ant' 'unzip' 'openssl' 'libcups' 'pam' 'patchelf' 'wget' 'yarn')
+optdepends=('git: for git support'
+            'subversion: for subversion support'
+            'openssh-askpass: for a git ssh access'
+            'quarto: for Quarto projects support')
+
+provides=('rstudio-desktop')
+conflicts=('rstudio-desktop' 'rstudio-desktop-bin' 'rstudio-desktop-preview' 'rstudio-desktop-git')
+source=("rstudio-$_pkgver.tar.gz::https://github.com/rstudio/rstudio/archive/refs/tags/v2023.06.2+561.tar.gz"
+        "https://github.com/quarto-dev/quarto/archive/refs/heads/release/rstudio-cherry-blossom.zip"
+        "https://nodejs.org/dist/v${_nodever}/node-v${_nodever}-linux-x64.tar.gz"
+        "qt.conf"
+        "pandoc_version.patch")
+
+sha256sums=('83469a4b7311d745c24d7d9e55c44aa4325dda3bdde50a903e2f9ead47d75cc7'
+            'df3a040f0cf4ce1892519082dd6822a8ca433e9e4f02d9394ab2931733f7e5a6'
+            '2c69e7b040c208b61ebf9735c63d2e5bcabfed32ef05a9b8dd5823489ea50d6b'
+            '723626bfe05dafa545e135e8e61a482df111f488583fef155301acc5ecbbf921'
+            '286925c442c1818979714feeec1577f03ae8a3527d2478b0f55238e2272a0b9e')
+
+noextract=("gin-${_ginver}.zip")
+
+# Retrives commit id of git tag associated with release  
+_get_commit_from_tag() {
+  git ls-remote https://github.com/rstudio/rstudio.git | grep v${pkgver} | sed 's/^\(.\{7\}\).*/\1/'
+}
+
+prepare() {
+    cd ${srcdir}/${_srcname}
+    # Do not use outdated version name of pandoc
+    patch -p1 < ${srcdir}/pandoc_version.patch
+
+    cd "${srcdir}/${_srcname}/dependencies/common"
+    install -d pandoc/${_pandocver}
+ 
+    ln -sfT /usr/share/myspell/dicts dictionaries
+    ln -sfT /usr/share/mathjax2 mathjax-27
+    ln -sfT /usr/bin/pandoc pandoc/${_pandocver}/pandoc
+
+    # Nodejs
+    install -d node/${_nodever}
+    cp -r "${srcdir}/node-v${_nodever}-linux-x64/"* node/${_nodever}
+    #cd "${srcdir}/${_srcname}/src/gwt/panmirror/src/editor"
+    # yarn config set ignore-engines true
+    #yarn install
+
+    # Fix links for src/cpp/session/CMakeLists.txt
+    cd "${srcdir}/${_srcname}/dependencies"
+    ln -sfT /usr/share/myspell/dicts dictionaries
+    ln -sfT /usr/share/mathjax2 mathjax-27
+    #ln -sfT /usr/bin/pandoc pandoc
+
+    # Panmirror is picked up now from Quarto repo
+    # Ideally: git clone --branch release/rstudio-cherry-blossom https://github.com/quarto-dev/quarto.git "${srcdir}/${_srcname}/src/gwt/lib/quarto"
+    mkdir -p "${srcdir}/${_srcname}/src/gwt/lib/quarto"
+    cp -r "${srcdir}/quarto-release-rstudio-cherry-blossom/"* "${srcdir}/${_srcname}/src/gwt/lib/quarto"
+}
+
+build() {
+
+    # Quarto
+    msg "Checking if Quarto is installed..."
```

Comment:
`msg` is an internal `makepkg` function, which shouldn't be used in PKGBUILDs. It can be replaced with `echo`.

Reply:
removed as quarto detection is surperfluous since we are not using quarto